src/utils/decorators/input/input.scss

```diff
@@ -195,7 +195,7 @@
   height: 15px;
   position: absolute;
   right: 5px;
-  top: 6px;
+  top: 28px;
```

Comment:
Regardless of the nature of that particular fix, it's still a common occurrence to see values like this pushed around by a few pixels because an icon is not aligned, those problems can be solved once and for all with flex